### DISCUSSION TITLE
Problem mit der Installation einer Go-e Wallbox

### DISCUSSION THREAD

**Discussion by RobToas** [camueller/SmartApplianceEnabler#272] [Installation und Betrieb]
Hallo zusammen,
Ich benutze SAE seit ein paar Monaten, und ich finde es wirklich gut - Vielen Danke an Camueller dafür!
Jetzt habe ich eine Go-e Wallbox gekauft. Ich habe sie in SAE installiert - so weit so gut.
Aber ich habe jetzt einen Fehler für die Installation in Sunny Home Manager.
(s. Screenshots)

Screenshots:
1/ in Sunny Home Manager
![Bild SHM](https://user-images.githubusercontent.com/106312337/170472355-91491ac2-e850-4d01-92f9-64cb16348a95.jpg)

2/ in SAE
![Bild SAE Wallbox](https://user-images.githubusercontent.com/106312337/170472360-1c6864d9-0e5b-46df-830a-a8bd6c68ba24.jpg)
![Bild SAE Status](https://user-images.githubusercontent.com/106312337/170472361-a1ff0a33-08d2-41d4-a03d-30ea9b2792ed.jpg)

Könnte mir jemand helfen?

Danke!
Rob

**Comment by camueller** [comment]
Vermutlich ist die ID F-00000001-100000000099-00 vergeben. Bitte die [Hinweise](https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Appliance_DE.md#id) beachten.

Vor der Änderung der ID das Gerät im Sunny Portal löschen. Im SAE ist das Ändern der ID aktuell ebenfalls nicht möglilch, d.h. Du musst auch dort das Gerät löschen und neu anlegen. Danach kannst Du es im Sunny Portal neu hinzufügen.

**Comment by RobToas** [comment]
Hallo,
Danke für die Antwort.
Ich habe 3 Male mit verschiedenen ID versucht, ich komme leider mit dem gleichen Ergebnis...
Gibt es noch eine andere Möglichkeit für den Fehler?
Danke!

**Comment by camueller** [comment]
Ich kann Dir leider nicht helfen bei diesem Problem. Entweder selbst nach einer Lösung suchen durch Ausprobieren oder bei SMA eine Suuportanfrage stellen.

**Comment by RobToas** [comment]
So ich habe noch etwas gesucht. Ich sehe in SHM log die Zeile "Das von Ihrem Sunny Home Manager gefundene Gerät mit der Seriennummer 987857340 ist bereits in einer anderen Anlage registriert und kann deshalb Ihrer Anlage nicht hinzugefügt werden."

Diese Zeile mit dem gleichen Inhalt ist mehrmals gekommen, obwohl ich die Prozedur wie oben geschrieben: "Vor der Änderung der ID das Gerät im Sunny Portal löschen. Im SAE ist das Ändern der ID aktuell ebenfalls nicht möglich, d.h. Du musst auch dort das Gerät löschen und neu anlegen. Danach kannst Du es im Sunny Portal neu hinzufügen."

->Es wäre als ob die Löschung und Neu Anlegung in SAE zur gleichen Seriennummer in SHM führen, obwohl der Inhalt in SAE nach der Neu Anlegung anders war.
Ist es normal?

Danke im Voraus für eure Unterstützung!
